backend/src/test/kotlin/org/darkSolace/muse/userModule/UserControllerApiTests.kt

```diff
@@ -92,19 +131,193 @@ class UserControllerApiTests {
         val response = restTemplate.getForEntity(url, Array<User>::class.java)
 
         Assertions.assertEquals(HttpStatus.OK, response.statusCode)
-        Assertions.assertTrue(response.body?.contains(user) ?: false)
-        Assertions.assertTrue(response.body?.contains(user2) ?: false)
-        Assertions.assertTrue(response.body?.contains(user3) ?: false)
-        Assertions.assertTrue(response.body?.contains(user4) ?: false)
+        Assertions.assertTrue(response.body?.contains(user?.toPublicUser()) ?: false)
+        Assertions.assertTrue(response.body?.contains(user2?.toPublicUser()) ?: false)
+        Assertions.assertTrue(response.body?.contains(user3?.toPublicUser()) ?: false)
+        Assertions.assertTrue(response.body?.contains(user4?.toPublicUser()) ?: false)
     }
 
     @Test
-    @Order(4)
+    @Order(5)
     fun getAll_Empty() {
         val url = generateUrl("/api/user/all")
         val response = restTemplate.getForEntity(url, Array<User>::class.java)
 
         Assertions.assertEquals(HttpStatus.OK, response.statusCode)
         Assertions.assertEquals(0, response.body?.size ?: 99)
     }
+
+    @Test
+    @Order(6)
+    fun suspendUser_success() {
+        //generate user to suspend
+        val signUpRequest = SignUpRequest("test", "1234", "test@example.com")
+        var user = userService.createUserFromSignUpRequest(signUpRequest)
+
+        //generate admin to perform the suspension
+        //create user and sign in
+        var url = generateUrl("/api/auth/signup")
+        restTemplate.postForEntity(
+            url,
+            SignUpRequest("test2", "123", "test2@example.com"),
+            String::class.java
+        )
+
+        userRoleService.changeRole(User(username = "test2", password = "", email = ""), Role.ADMINISTRATOR)
+
+        url = generateUrl("/api/auth/signin")
+        val signInResponse = restTemplate.postForEntity(
+            url,
+            SignUpRequest("test2", "123", "test2@example.com"),
+            JwtResponse::class.java
+        )
+
+        //suspend user via api
+        url = generateUrl("/api/user/suspend/${user?.id}")
```

Comment:
did you not change this to suspend/confirmationCode/something?

Reply:
suspending a user is done via `/api/user/suspend/<id>` - if a user confirms it suspension (after reviewing the warning letter) `/api/user/suspend/confirm/<uuid>` is called